flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CalcITCase.scala

```diff
@@ -548,4 +548,25 @@ class CalcITCase extends StreamingTestBase {
     TestBaseUtils.compareResultAsText(result, "42")
   }
 
+  @Test
+  def testExecuteInsertToTableDescriptor(): Unit = {
```

Comment:
I think it is fine to use an ITCase here because we also trigger an execution but `Calc` is not a good location to find it again, maybe `org.apache.flink.table.planner.runtime.stream.table.TableSinkITCase`?